File: src/main/java/org/taktik/icure/entities/IdentityDocumentReader.java
package org.taktik.icure.entities;

public class IdentityDocumentReader {

    private String justificatifDocumentNumber;
    private String SupportSerialNumber;
    private Long timeReadingEIdDocument;
    private int eIdDocumentSupportType;
    private int reasonManualEncoding;
    private int reasonUsingVignette;

    public String getJustificatifDocumentNumber() {
        return justificatifDocumentNumber;
    }

    public void setJustificatifDocumentNumber(String justificatifDocumentNumber) {
        this.justificatifDocumentNumber = justificatifDocumentNumber;
    }

    public String getSupportSerialNumber() {
        return SupportSerialNumber;
    }

    public void setSupportSerialNumber(String supportSerialNumber) {
        SupportSerialNumber = supportSerialNumber;
    }

    public Long getTimeReadingEIdDocument() {
        return timeReadingEIdDocument;
    }

    public void setTimeReadingEIdDocument(Long timeReadingEIdDocument) {
        this.timeReadingEIdDocument = timeReadingEIdDocument;
    }

    public int geteIdDocumentSupportType() {
        return eIdDocumentSupportType;
    }

    public void seteIdDocumentSupportType(int eIdDocumentSupportType) {
        this.eIdDocumentSupportType = eIdDocumentSupportType;
    }

    public int getReasonManualEncoding() {
        return reasonManualEncoding;
    }

    public void setReasonManualEncoding(int reasonManualEncoding) {
        this.reasonManualEncoding = reasonManualEncoding;
    }

    public int getReasonUsingVignette() {
        return reasonUsingVignette;
    }

    public void setReasonUsingVignette(int reasonUsingVignette) {
        this.reasonUsingVignette = reasonUsingVignette;
    }
}

File: src/main/kotlin/org/taktik/icure/entities/embed/IdentityDocumentReader.kt
package org.taktik.icure.entities.embed

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude
import com.github.pozo.KotlinBuilder
import java.io.Serializable

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
@KotlinBuilder
data class IdentityDocumentReader(
        val justificatifDocumentNumber: String? = null,
        val supportSerialNumber: String? = null,
        val timeReadingEIdDocument: Long? = null,
        val eidDocumentSupportType: Int = 0,
        val reasonManualEncoding: Int = 0,
        val reasonUsingVignette: Int = 0
) : Serializable
